backend/inspirehep/editor/authorlist_utils.py

```diff
@@ -405,3 +406,128 @@ def authorlist(text):
             )
         result["authors"] = builder.record["authors"]
     return result
+
+
+class GrobidAuthors(object):
```

Comment:
I would move these two classes to https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/matcher/parsers.py